wxfs/database/demo/database.py

```diff
@@ -71,26 +71,25 @@ def populate(session):
     session.commit()
 
 
-def main(dsn, action):
+def main(dsn, actions):
     """
-    Populate a wxfs database with some stuff. Yeah, man!
+    Create a wxfs database and/or populate with some stuff. Yeah, man!
 
     :param dsn: connection info for the modelmeta database to update
+    :param actions: List of actions... Can be "create", "populate", or a list of both
     """
     engine = create_engine(dsn)
 
     engine.execute("SET search_path = wxfs, public")
 
-    if action == "create":
+    if "create" in actions:
         create_database(engine)
-        return
 
-    Session = sessionmaker(bind=engine)
-    session = Session()
-
-    populate(session)
-
-    session.close()
+    if "populate" in actions:
+        Session = sessionmaker(bind=engine)
+        session = Session()
+        populate(session)
+        session.close()
```

Comment:
I know it's a bit lame of me to ask for explanations in this singularly undocumented code, but maybe a note about usage and what `populate` does?